src/app/color-legend/color-legend.component.scss

```diff
@@ -0,0 +1,25 @@
+//  @Arnaud: this class is duplicated in contribution-graph component.
```

Comment:
Good question, I am not sure myself how to avoid this type of duplication. I'm sure there's a better way we can find. 

One solution would be to put (or import) the css in style.css, but that may be a bit too general?